### ISSUE TITLE
NTR: N-carbamoyl-beta-alanine present in urine (Lab: chemistry: amino compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8495]
New term request N-carbamoyl-beta-alanine present in urine (segal_030501201435):

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000279
- label: Elevated urinary N-carbamoyl-beta-alanine level
- parents: [Increased urinary non-proteinogenic amino acid level (HP:0033098)](https://hpo.jax.org/app/browse/term/HP:0033098)
- pmid: 
- def: The amount of N-carbamoyl-beta-alanine present in urine, when adjusted for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8495
- synonyms:
